content/docs/10-audit-logs/05-kube-api-audit-logging.md

```diff
@@ -0,0 +1,209 @@
+The Kubernetes API parameters are located toward the top of the file. The code snippet below showcases where in the file to place the parameters.
+
+  <br />
+
+  ```yaml
+  spec:
+    containers:
+    - command:
+      - kube-apiserver
+      - --advertise-address=172.18.0.2
+      - --allow-privileged=true
+      - --authorization-mode=Node,RBAC
+  ```
+
+8. Go ahead and add the following audit parameters under the `- kube-apiserver` line.
+
+  <br />
+
+  ```shell
+      - --audit-policy-file=/etc/kubernetes/audit-policy.yaml
+      - --audit-log-path=/var/log/kubernetes/audit/audit.log
+      - --audit-log-batch-max-size=5
+      - --audit-log-compress
+      - --audit-log-format=json
+      - --audit-log-maxage=30
+      - --audit-log-maxbackup=100
+      - --audit-log-maxsize=50
+      - --audit-log-mode=batch
+      - --audit-log-truncate-enabled
+      - --audit-log-truncate-max-batch-size=10485760
+      - --audit-log-truncate-max-event-size=102400
+      - --audit-log-version=audit.k8s.io/v1
+  ```
+
+9. Save your changes and exit the file. When you exit the file, the changes will automatically get picked up by the Kubelet process and applied.
+
+You can also add the following Kubernetes API parameters to fine-tune the audit logging.
+
+| Parameter                         | Type     | Description                                                                                                                   |
+|-----------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------|
+| -`-audit-log-batch-max-wait`        | duration | The amount of time to wait before force writing the batch that hadn't reached the max size. Only used in batch mode. Ex: `"5s"` |
+| `--audit-log-batch-throttle-enable` |   boolean       | Whether batching throttling is enabled. Only used in batch mode.                                                              |
+| `--audit-log-batch-throttle-qps`    | float    | The maximum average number of batches per second. Only used in batch mode                                                     |
+
+
+To learn more about each of the Kubernetes API server flags, visit the Kubernetes API parameter [documentation page](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
+
+# Validate
+
+You can validate that audit logs are captured by navigating to the specified audit folder in the `--audit-log-path` parameter. 
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Identify one of your cluster contro-plane nodes. You find a cluster node by navigating to the left **Main Menu** and selecting **Clusters**. Click on your cluster to access the details pages and click on the **Nodes** tab. The tab contains information about each pool, select a node from the **Master Pool** to view its IP address.
```

Comment:
Click on your cluster to access the details pages, and click on the **Nodes** tab. The tab contains information about each pool; select a node from the **Master Pool** to view its IP address.